src/main.js

```diff
@@ -0,0 +1,139 @@
+const TOHBoard = function () {
+  let board = [[5, 4, 3, 2, 1], [], []];
+  const boardObject = {
+    1: board[0],
+    2: board[1],
+    3: board[2],
+  };
+
+  const initialDisplay = function () {
+    board.map((peg) => console.log(`--- ${peg}`));
+  };
+
+  const displayBoard = function () {
+    console.log("Current Board:");
+    for (const peg in boardObject) {
+      console.log(`${peg}: --- ${boardObject[peg]}`);
+    }
+    return board;
+  };
+
+  const moveDisc = function (initialPeg, newPeg) {
+    const initialArray = boardObject[initialPeg];
+    const newArray = boardObject[newPeg];
```

Comment:
not an issue here but objects are passed by reference and not value, this approach can create bugs in the future. You need to find a way to 'clone' the object for example with the spread operator (...), that creates a copy and its safer for the future.